iree/compiler/Dialect/HAL/Target/VulkanSPIRV/VulkanSPIRVTarget.cpp

```diff
@@ -153,10 +153,12 @@ class VulkanSPIRVTargetBackend : public SPIRVTargetBackend {
     iree_SpirVExecutableDef_end_as_root(builder);
 
     // Add the binary data to the target executable.
-    executableBuilder.create<IREE::HAL::ExecutableBinaryOp>(
+    auto binaryOp = executableBuilder.create<IREE::HAL::ExecutableBinaryOp>(
         targetOp.getLoc(), targetOp.sym_name(),
         executableBuilder.getStringAttr("SPVE"),
         builder.getBufferAttr(executableBuilder.getContext()));
+    binaryOp.mime_typeAttr(
+        executableBuilder.getStringAttr("application/x-flatbuffers"));
```

Comment:
We had a user wanting to inspect Vulkan/SPIR-V outputs for a compiled model. I'd like to have a simple set of instructions like those at https://www.tensorflow.org/xla#inspect_compiled_programs for how to do that. There are all sorts of places we _could_ dump IR, but this (the zip output from this PR) looks closer to something we could default to pointing users at. I wouldn't want to ask users to run a compiler tool with special flags (`-print-ir-after-all`) and parse through the output ("look after this specific pass"), but asking them to look in a zip file or output directory isn't so bad.

Reply:
For sure! That was my hope: once we get rid of the flatbuffer wrapper around SPIRV blobs (and CUDA/etc) you'll be able to open up the flatbuffers as zips and just extract the .spv files. You can then run them through whatever disassembly/decompiler tool you want. All this works from the command-line too of course with `unzip` and other tools, so you could even setup a "dump and disassemble all shaders in this module" (or ELFs, etc).

Reply:
(#5662 is tracking removing the flatbuffer wrappers)